octobot_trading/enums.py

```diff
@@ -355,11 +366,12 @@ class ExchangeConstantsPositionColumns(enum.Enum):
     MARK_PRICE = "mark_price"
     LIQUIDATION_PRICE = "liquidation_price"
     BANKRUPTCY_PRICE = "bankruptcy_price"
-    UNREALIZED_PNL = "unrealised_pnl"   # unrealised_pnl on Bybit api and unrealized_pnl on exchange UI
-    REALISED_PNL = "realised_pnl"
+    UNREALIZED_PNL = "unrealized_pnl"  # unrealized_pnl on Bybit api and unrealized_pnl on exchange UI
+    REALIZED_PNL = "realized_pnl"
     CLOSING_FEE = "closing_fee"
     QUANTITY = "quantity"
     SIZE = "size"
+    VALUE = "value"
```

Comment:
Is there a difference between between "value" and "notional"?

Reply:
Oh its the same I fixed it